File: src/main/java/com/dv/telegram/google/WikiBotGoogleSheet.java
package com.dv.telegram.google;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class WikiBotGoogleSheet { // only contains strings from the Google Sheet, not semantically parsed
    private SheetData wikiPagesSheet;
    private SheetData cityChatsSheet;
    private SheetData commandsSheet;
}

File: src/main/java/com/dv/telegram/google/WikiBotGoogleSheet.kt
package com.dv.telegram.google

data class WikiBotGoogleSheet( // only contains strings from the Google Sheet, not semantically parsed
    val wikiPagesSheet: SheetData,
    val cityChatsSheet: SheetData,
    val countryChatsSheet: SheetData,
    val commandsSheet: SheetData
)
